### PR TITLE
Improve Docs Design

### PR DESCRIPTION
**Pull request by dev0Guy**
## Changes:
- Slide bar tabs were removed and moved into navigation bar.
- Changed primary Color to teal , on mouse hover (accent) change to amber (to match logo colors)
- Changed theme type (dark, light) according to system preference. 
- Added functionality of switching between dark & light color theme 

**Before**
Screenshot 2024-12-23 at 12 34 22

**After Dark**
Screenshot 2024-12-23 at 12 37 29

**After Light**
Screenshot 2024-12-23 at 12 37 45

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -9,23 +9,27 @@ repo_name: Pynest
...
+      scheme: default
+      primary: teal
+      accent: amber
       toggle:
         icon: material/toggle-switch
+        name: Switch to dark mode
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
```
[the remark this answers is not in the document]
Comment (dev0Guy) [reply mkdocs.yml:29]: Hi, 
Here is the [linkedin](https://www.linkedin.com/in/guy-arieli-4127b8198/)
Additionally, The color is taken from mkdocs material [options](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/). 
If you prefer a difference color I would gladly change.

[the remark this answers is not in the document]
Comment (dev0Guy) [reply mkdocs.yml]: Solved this by changing into blue grey. 
Screenshot 2024-12-24 at 11 22 36
in commit [a554596](https://github.com/PythonNest/PyNest/pull/91/commits/a5545968aac49fde3479a393b57f6546ea7793e0)

[the remark this answers is not in the document]
Comment (dev0Guy) [reply mkdocs.yml]: **Dark**
Screenshot 2024-12-24 at 11 26 08

[the remark this answers is not in the document]
Comment (itay-dar-lmnd) [reply mkdocs.yml]: I'm sorry for the troubles, but after second though, the first pallete looks better and gives people something familiar when they are on pynest site. so it was a good idea from the beginning. could you change that back?

[the remark this answers is not in the document]
Comment (dev0Guy) [reply mkdocs.yml]: sure, changing it back

[the remark this answers is not in the document]
Comment (dev0Guy) [reply mkdocs.yml:29]: fixed

**Review by ItayTheDar — APPROVED** [review]

**Pull request merged**